oak-core/src/main/java/org/apache/jackrabbit/oak/security/user/UserConfigurationImpl.java

```diff
@@ -176,6 +176,13 @@
                 name = "RFC7613 Username Comparison Profile",
                 description = "Enable the UsercaseMappedProfile defined in RFC7613 for username comparison.")
         boolean enableRFC7613UsercaseMappedProfile() default false;
+
+        // for OAK-9675
+        @AttributeDefinition(
+                name = "Authorizable Properties Mixin Types",
+                description = "Optional configuration defining the names of additional " +
```

Comment:
the 'additional' strikes me here..... my reading of the improvement request (as pointed out before) would be to limit authorizable properties to those defined by the configured mixin types and not allow them 'in addition'.

as by this comment:
`The proposed improvement here is to add an optional configuration property that would define the names of mixin types that are allowed to define authorizable properties.  Any property definition defined by a mixin type in this set would be included, and anything else would be excluded as before`.

Reply:
Fair enough, I'll change "additional" to "any" to provide better clarity.